app/grandchallenge/cases/models.py

```diff
@@ -325,6 +326,32 @@ def image_file_path(instance, filename):
     )
 
 
+class DICOMImageSet(UUIDModel):
+    image_set_id = models.CharField(
+        max_length=32,
+        unique=True,
+        help_text="The ID of the image set in AWS Health Imaging.",
+    )
+    dicom_image_set_upload = models.OneToOneField(
+        to="DICOMImageSetUpload",
+        editable=False,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="dicom_image_set",
+    )
+
+
+@receiver(post_delete, sender=DICOMImageSet)
+def delete_image_set(*_, instance: DICOMImageSet, **__):
+    from grandchallenge.cases.tasks import delete_healthimaging_image_set
+
+    on_commit(
+        delete_healthimaging_image_set.signature(
+            kwargs=dict(image_set_id=instance.image_set_id)
```

Comment:
The style in the codebase is to use these constructions for dictionaries, is there a reason to change?

```suggestion
            kwargs={"image_set_id": instance.image_set_id}
```